rules/windows/powershell/powershell_CL_Mutexverifiers_LOLScript.yml

```diff
@@ -0,0 +1,35 @@
+title: Execution via CL_Mutexverifiers.ps1
+id: 39776c99-1c7b-4ba0-b5aa-641525eee1a4
+description: Detects Execution via runAfterCancelProcess in CL_Mutexverifiers.ps1 module
+status: experimental
+author: oscd.community, Natalia Shornikova
+date: 2020/10/14
+references:
+    - https://github.com/LOLBAS-Project/LOLBAS/blob/master/yml/OSScripts/CL_mutexverifiers.yml
+    - https://twitter.com/pabraeken/status/995111125447577600
+tags:
+    - attack.defense_evasion
+    - attack.t1216
+logsource:
+    product: windows
+    service: powershell
+detection:
+    selection:
+        EventID: 4104
+        ScriptBlockText|contains|all:
+          - 'CL_Mutexverifiers.ps1'
+          - 'runAfterCancelProcess'
+    selection2:
+        EventID: 4104
+        ScriptBlockText|contains:
+          - 'CL_Mutexverifiers.ps1'
+          - 'runAfterCancelProcess'
+    timeframe: 1m
+    condition: 
+        - selection
+      # PS > Import-Module c:\Windows\diagnostics\system\Audio\CL_Mutexverifiers.ps1; runAfterCancelProcess c:\Evil.exe
+        - selection2 | count(ScriptBlockText) by Computer > 2
```

Comment:
It seems that it's very similar to the https://github.com/Neo23x0/sigma/pull/1184
Could you please adjust it?

Reply:
Hello @nsaddler! Would you like to proceed with working on this rule?

Reply:
Sorry for delay!
Of course I would, I did the same as in the #1184 (splitting into two rules). I think these conditions are equivalent, an adversary can enter it as by one line so by two ones.

Reply:
Thanks!